### PR TITLE
Add tooltips for visualisations

### PR DESCRIPTION
**Pull request by caugello**

### CODE REVIEW

File: src/PageViewCard.tsx [hunk 1]
```diff
@@ -19,7 +20,20 @@ export class PageViewCard extends React.Component<IPageViewCardProps> {
...
+            <FlexItem>
+              <Title size="lg">Repository overview page views</Title>
+            </FlexItem>
+            <FlexItem>
+              <Tooltip
+                className="rh-page-views-card-tooltip"
+                position={TooltipPosition.top}
+                content="Displays daily views of the repository and any detail pages in the Container Catalog."
+              >
+              <InfoCircleIcon color={'#8BC1F7'}/>
```
Comment (amisstea) [inline src/PageViewCard.tsx]: Is this a color from the patternfly color pallete?

Comment (amisstea) [reply src/PageViewCard.tsx]: Also, please indent over one level.

Comment (kartouch) [reply src/PageViewCard.tsx]: Fixed indentation and using color pallete.

**Comment by amisstea** [comment]
LGTM :+1:

**Pull request merged**